public/pages/workflow_detail/workflow_inputs/processor_inputs/modals/override_query_modal.tsx

```diff
@@ -0,0 +1,263 @@
+              <EuiPopover
+                button={
+                  <EuiSmallButton
+                    onClick={() => setPresetsPopoverOpen(!presetsPopoverOpen)}
+                  >
+                    Choose from a preset
+                  </EuiSmallButton>
+                }
+                isOpen={presetsPopoverOpen}
+                closePopover={() => setPresetsPopoverOpen(false)}
+                anchorPosition="downLeft"
+              >
+                <EuiContextMenu
+                  initialPanelId={0}
+                  panels={[
+                    {
+                      id: 0,
+                      items: QUERY_PRESETS.map((preset: QueryPreset) => ({
+                        name: preset.name,
+                        onClick: () => {
+                          try {
+                            setFieldValue(
+                              queryFieldPath,
+                              preset.query
+                                // sanitize the query preset string into valid template placeholder format, for
+                                // any placeholder values in the query.
+                                // for example, replacing `"{{vector}}"` with `${vector}`
+                                .replace(
+                                  new RegExp(`"${VECTOR_FIELD_PATTERN}"`, 'g'),
+                                  `\$\{vector_field\}`
+                                )
+                                .replace(
+                                  new RegExp(`"${VECTOR_PATTERN}"`, 'g'),
+                                  `\$\{vector\}`
+                                )
+                                .replace(
+                                  new RegExp(`"${TEXT_FIELD_PATTERN}"`, 'g'),
+                                  `\$\{text_field\}`
+                                )
+                                .replace(
+                                  new RegExp(`"${IMAGE_FIELD_PATTERN}"`, 'g'),
+                                  `\$\{image_field\}`
+                                )
+                                .replace(
+                                  new RegExp(`"${LABEL_FIELD_PATTERN}"`, 'g'),
+                                  `\$\{label_field\}`
+                                )
+                                .replace(
+                                  new RegExp(`"${QUERY_TEXT_PATTERN}"`, 'g'),
+                                  `\$\{query_text\}`
+                                )
+                                .replace(
+                                  new RegExp(`"${QUERY_IMAGE_PATTERN}"`, 'g'),
+                                  `\$\{query_image\}`
+                                )
+                                .replace(
+                                  new RegExp(`"${MODEL_ID_PATTERN}"`, 'g'),
+                                  `\$\{model_id\}`
+                                )
+                            );
+                          } catch {}
```

Comment:
Can we have a failure message within catch?

Reply:
Copied this over, and realized we don't need it, since we aren't doing any actions that can throw exceptions. Removed it altogether.